### ISSUE TITLE
Support Docker release candidates

### ISSUE THREAD

**Issue by collinpeters** [howtowhale/dvm#140]
I'm eager to try out dvm but at the moment my use case is a need to test Docker 1.13.0 rc1 however I don't see it in the list of options. Would be nice to support release candidates.

**Comment by carolynvs** [comment]
You are in luck! 😄 

```
$ dvm install experimental
Installing experimental...
Now using Docker experimental

$ docker --version
Docker version 1.13.0-rc1, build 75fd88b
```

I'll leave this open as it would be nice to make this feature more visible. Maybe including experimental in `dvm ls-remote` would help?

**Comment by carolynvs** [comment]
This is fixed in #156.
